pytentiostat/routines.py

```diff
@@ -38,14 +38,16 @@ def _initialize_arduino(com):
     Creates board object with Arduino(). If the connection fails it prints an error message and exits.
     Parameters
     ----------
-    com: string
         the COM port that the potentiostat is connected to.
     """
     try:
         board = Arduino(com,
                         baudrate=_BAUD_RATE)  # opens communication to Arduino
-        print("Pytentiostat connected to {}.\n".format(
+        if board.firmware == 'pytentiostat_firmata.ino':
+            print("Pytentiostat connected to {}.\n".format(
             com))
+        else:
+            print('the board is not a Pytentiostat')
```

Comment:
Ah okay this is good way to check if it is a pytentiostat.

Reply:
Yes, this is the true change.

Reply:
This is a good addition 👍 Right now you are working on the code that runs entirely in the command line but we have another version in the GUI folder that has a GUI and works differently. I think it would be nice if you could implement this functionality there as well.